### DISCUSSION TITLE
Fill a field from lookup

### DISCUSSION THREAD

**Discussion by dmontilla** [serenity-is/Serenity#6701] [Help] [answered]
is this not supposed to fill automatically? 

![image](https://user-images.githubusercontent.com/42835912/220753875-20092609-40e8-4123-89d9-fc3cb8534e76.png)

![image](https://user-images.githubusercontent.com/42835912/220753893-394c5af0-d874-46f6-9b68-d11a921fc4fd.png)

![image](https://user-images.githubusercontent.com/42835912/220754039-7f0f07f9-8d8c-49bf-aa24-2c1a37e5a48a.png)

by the way I was trying to open a new discussion on https://github.com/serenity-is/Serenity/discussions but I couldn´t find a way to mention you there as you are pretty fast on response

**Comment by arunkumar12712** [comment, marked as answer]
Try this in Dialog   

     protected afterLoadEntity() {
            super.afterLoadEntity();
this.form.RoomNoId.changeSelect2(()=>{
   RoomService.Retrieve({
   EntityId:this.form.RoomNo.value
  }, res=>{
    this.form.rent.value = res.entity.rent
 })
})
}

**Reply by dmontilla** [reply]
@arunkumar12712 thanks, that work
